tools/autograd/gen_python_functions.py

```diff
@@ -271,7 +271,8 @@ def parsed_arg_expr(arg, arg_index):
 
     unpack = UNPACK_METHODS.get(typename)
     if unpack is None:
-        raise RuntimeError('type \'{}\' is not supported'.format(typename))
+        raise RuntimeError('type \'{}\' is not supported (found in {})'
+                           .format(typename, op_name))
```

Comment:
If you're back in here for some other reason before you land, can you add the "found in {op_name}" to the other RuntimeErrors in this function? 😬